pkg/hub/manager.go

```diff
@@ -74,6 +81,15 @@ func RunControllerManager(ctx context.Context, controllerContext *controllercmd.
 		controllerContext.EventRecorder,
 	)
 
+	managedClusterDeletionController := managedcluster.NewManagedClusterDeletionController(
+		kubeClient,
+		metaClient,
+		clusterClient,
+		clusterInformers.Cluster().V1().ManagedClusters(),
+		[]schema.GroupVersionResource{},
```

Comment:
we may consider to export this as a controller option, right?

Reply:
yes, this should be under a feature gate also